scripts/in_container/check_environment.sh

```diff
@@ -28,50 +28,16 @@ EXIT_CODE=0
 # Even if this message might be harmless, it might hide the real reason for the problem
 # Which is the long time needed to start some services, seeing this message might be totally misleading
 # when you try to analyse the problem, that's why it's best to avoid it,
-function run_nc() {
```

Comment:
@jscheffl i looked at some scripts already have this for connectivity check. so i feel we can re-use making these as a separate functions. so moved these two functions inside `scripts/in_container/check_connectivity.sh`.  and we can use this script.

Now i have called these functions to check the web-server connectivity. WDYT is that fine? :)